Makefile

```diff
@@ -8,14 +8,14 @@ include CREDENTIALS
 
 .PHONY: build_only
 build_only:
-	-rm ./build/*.xpi
-	web-ext build --source-dir=src --artifacts-dir=build
-	mv build/wayback_machine-$(BUILD_VERSION).xpi build/Wayback_Machine_Firefox_V$(BUILD_VERSION).xpi
-	git add *.xpi
+	-rm ./build/*.zip
```

Comment:
I believe that somewhere around **web-ext@1.2.0** they switched from .XPI to .ZIP by default.